include/BDSFieldEMAcceleration.hh

```diff
@@ -0,0 +1,28 @@
+//
+// Created by Boogert Stewart on 16/08/2024.
+//
+
+#ifndef BDSIM_BDSFIELDEMACCELERATION_HH
+#define BDSIM_BDSFIELDEMACCELERATION_HH
+
+#include <vector>
```

Comment:
Pedantic 💯  ordering for codebase style:

bdsim includes

geant4 includes

cpp general includes